keras_cv/layers/regularization/squeeze_excite.py

```diff
@@ -12,13 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import tensorflow as tf
-from tensorflow import keras
-from tensorflow.keras import layers
+from keras_cv.backend import keras
+from keras_cv.backend import ops
 
 
 @keras.utils.register_keras_serializable(package="keras_cv")
-class SqueezeAndExcite2D(layers.Layer):
+class SqueezeAndExcite2D(keras.layers.Layer):
```

Comment:
If we're updating this, our `RegNet` legacy model will have to be ported at the same time as it depends on this layer.

Our base CI doesn't run the `RegNet` test, but `gcbrun` will, and this will cause it to fail (I had to do the same thing for a legacy model in #4 so feel free to merge past that and copy the approach I used there)

Reply:
Done. Thanks for letting me know!